bin/packages/build-types.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable eslint-comments/disable-enable-pair, no-console */
+
+/**
+ * External dependencies
+ */
+const execa = require( 'execa' );
+const glob = require( 'fast-glob' );
+const path = require( 'path' );
+
+const tscPath = path.resolve(
+	__dirname,
+	'..',
+	'..',
+	'node_modules',
+	'.bin',
+	'tsc'
+);
+
+async function main() {
+	const packagesWithTs = await glob( 'packages/*/tsconfig.json', {
+		cwd: path.resolve( __dirname, '..', '..' ),
+	} );
+	const projectPaths = packagesWithTs.map( ( tsconfigPath ) =>
+		path.resolve( path.dirname( tsconfigPath ) )
```

Comment:
Is the extra `resolve` needed? Shouldn't `dirname` do the right thing already?

```suggestion
		path.dirname( tsconfigPath )
```